### PR TITLE
Add code of conduct and contributing files

### PR DESCRIPTION
**Pull request by julioxavierr**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Development
+
+Visit the [Issue tracker](https://github.com/reduxjs/redux-toolkit/issues) to find a list of open issues that need attention.
+
+Fork, then clone the repo:
+```
+git clone https://github.com/your-username/redux-toolkit.git
+```
+
+### Building
```
Comment (markerikson) [inline CONTRIBUTING.md]: I think this section needs to be updated to match the actual build scripts we have.  We don't have separate commands for different module types - we have one `build` command that generates everything.

Comment (julioxavierr) [reply CONTRIBUTING.md]: Done. It still create CommonJS and UMD?

Comment (markerikson) [reply CONTRIBUTING.md]: Yeah, you can see we pass those args to TSDX.

**Comment by markerikson** [comment]
Also, the Travis build is failing because both files need to be formatted with Prettier.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+We are open to, and grateful for, any contributions made by the community. By contributing to Redux Toolkit, you agree to abide by the [code of conduct](https://github.com/reduxjs/redux-toolkit/blob/master/CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+
+Before opening an issue, please search the [issue tracker](https://github.com/reduxjs/redux-toolkit/issues) to make sure your issue hasn't already been reported.
+
+Please ask any general and implementation specific questions on [Stack Overflow with a Redux tag](http://stackoverflow.com/questions/tagged/redux?sort=votes&pageSize=50) for support.
```
Comment (phryneas) [inline CONTRIBUTING.md]: Maybe a redux-toolkit tag - or both - instead? I know I've seen that used over there.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Please open an issue with a proposal for a new feature or refactoring before starting on the work. We don't want you to waste your efforts on a pull request that we won't want to accept.
+
+## Submitting Changes
+
+- Open a new issue in the [Issue tracker](https://github.com/reduxjs/redux-toolkit/issues).
+- Fork the repo.
+- Create a new feature branch based off the `master` branch.
+- Make sure all tests pass and there are no linting errors.
+- Submit a pull request, referencing any issues it addresses.
```
Comment (phryneas) [inline CONTRIBUTING.md]: - If you changed external-facing types, make sure to also build the project locally and include the updated [etc/redux-toolkit.api.md](https://github.com/reduxjs/redux-toolkit/blob/master/etc/redux-toolkit.api.md) in your pull request.

**Comment by markerikson** [comment]
Looks like that should work.  Thanks!

**Pull request merged**